MDS_3/test/MDS_3/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Created by the script cgal_create_cmake_script
+# This is the CMake script for compiling a CGAL application.
+
+cmake_minimum_required(VERSION 3.1...3.15)
+project( MDS_3_Tests )
+
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
+find_package(CGAL REQUIRED)
+
+# Use Eigen
+find_package(Eigen3 3.1.0 REQUIRED) #(3.1.0 or greater)
+include(CGAL_Eigen3_support)
+if (NOT TARGET CGAL::Eigen3_support)
+  message(STATUS "This project requires the Eigen library, and will not be compiled.")
+  return()
+endif()
```

Comment:
I think that can be removed, because you have added `REQUIRED` to the `find_package` for `Eigen3`.
```suggestion
```